components/Api/JsonLink/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { FormattedMessage } from 'react-intl';
+import styles from './index.module.scss';
+import type { FC } from 'react';
+
+type JsonLinkProps = {
+  fileName: string;
+  version: string;
+};
+
+const JsonLink: FC<JsonLinkProps> = ({ fileName, version }) => (
+  <div className={styles.json}>
+    <a
+      href={`https://nodejs.org/docs/latest-${version}.x/api/${fileName}.json`}
+    >
+      <FormattedMessage id="components.api.jsonLink.title" tagName="span" />
```

Comment:
The icon is missing

Reply:
try use that
```suggestion
      <FormattedMessage id="components.api.jsonLink.title" tagName="span" /><FaRobot />
```
and import it like that:
```js
import { FaRobot } from 'react-icons/fa';
```

Reply:
Wasn't sure if it should be moved but this was helpful and have now added the robot icon - thanks!

Reply:
Pretty much what @AugustinMauroy mentioned, @Ve33y, you can use the package we already gave (`react-icons`) for the Rbot Icon :)